### DISCUSSION TITLE
Scene failed to load custom client localhost

### DISCUSSION THREAD

**Discussion by dbsjin1** [mozilla/hubs#4888] [Hubs Cloud Questions] [answered]
**Description**
I have a working cloud instance, everything is working as expected on the website, the issue is getting a development environment.
Following the doc: 
https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html

**To Reproduce**
1. Follow this tutorial https://hubs.mozilla.com/docs/hubs-cloud-custom-clients.html
2. After npm start visit :  https://localhost:8080/
3. Click create room
4. Get The scene failed to load.
5. Console first error: Failed to load resource: the server responded with a status of 404 (Not Found)

**Expected behavior**
Getting a room as on the live website with localhost and be able to develop on it

**Screenshots**
![image](https://user-images.githubusercontent.com/93541183/140411464-a7e59f63-250b-4c66-bc88-9d80e9f9ff31.png)

**Hardware** 
 - Device: [Desktop]
 - OS: [ Windows 10]
 - Browser: [Chrome]

**Console**
GET https://localhost:8080/manifest.webmanifest 404 (Not Found)
manifest.webmanifest:1 Manifest: Line: 1, column: 1, Syntax error.
An SSL certificate error occurred when fetching the script.
hub.js:954 Reticulum @ determined-cleric.minikosm.com: v1.0.20210618144438 on minikosm-01
phoenix-utils.js:125 Phoenix Socket URL: wss://determined-cleric.minikosm.com
hub.js:1086 Logged into account 1092072784852418562
hub.js:505 Dialog host: determined-cleric.minikosm.com:8443
hub.js:379 Scene URL: https://192.168.1.127:8080/cors-proxy/https://minikosm-01-assets.minikosm.com/files/27453223-f86a-422d-b128-93e19fa845ac.bin
gltf-model-plus.js:42 Releasing uncached gltf https://192.168.1.127:8080/cors-proxy/https://minikosm-01-assets.minikosm.com/files/27453223-f86a-422d-b128-93e19fa845ac.bin

**Additionally**
The quick start on github doesn t work for me.

Thanks!

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/HUBS-1141)

**Comment by ErikOrjehag** [comment]
I have the same issue

**Comment by matthewbcool** [comment]
`npm ci`

**Running against HC:**
ensure you have ran `npm run login` and have successfully logged in
`npm start`
**you may need to delete your local storage**

**For testing against the hubs dev server:**
Try setting up hubs local hostname:
https://github.com/mozilla/reticulum#1-setup-the-hubslocal-hostname

ensure that your .env file contains `CORS_PROXY_SERVER="hubs-proxy.com"`
then use:
`npm run dev`
and go to: https://hubs.local:8080/

**Comment by georgebutler** [comment]
I'm facing the same issue, the fix didn't seem to work for me.

**Reply by matthewbcool** [reply]
updated that answer, let me know if that is helpful. thanks!

**Reply by nupur711** [reply]
I am facing the same issue. Any solution?

**Comment by ErikOrjehag** [comment]
I think logging in solved it for me, not sure why... Both admin and non admin accounts work but I have to be logged in.

**Comment by esperecyan** [comment, marked as answer]
I was able to work around this bug with https://github.com/mozilla/hubs/issues/4194#issuecomment-828881267.
Create .env file and write `HOST_IP=hubs.local`.

**Reply by esperecyan** [reply]
Until the other day I was able to connect with `HOST_IP=hubs.local`, but now when I run `npm run dev`, I have to write `CORS_PROXY_SERVER=hubs-proxy.com` instead or the scene will not load successfully.

Also, `npm run dev` used to load the same rooms and avatars as the login destination, but now it is unrelated to the login destination, just like `npm start`.

**Comment by manuareraa** [comment]
For me the error was - SSL certificate. So, if you can just open the link for which it shows an SSL error in a new tab and click on proceed, you will be able to load the scene when you refresh the page.

**Comment by albirrkarim** [comment]
i hope my repo can help you https://github.com/albirrkarim/mozilla-hubs-installation-detailed

**Reply by alessiashen** [reply]
I have the same problem，can you tell me what to do？

**Comment by alessiashen** [comment]
I have the same problem，can you tell me what to do？
